nodes/config/tts.html

```diff
@@ -48,22 +13,20 @@
     },
     oneditprepare: function() {
       var node = this;
-      languages.forEach(function(language) {
+      for (var languageCode in languages) {
         $('#node-config-input-language').append(
           $('<option>', {
-            value: language.code,
-            text: '[' + language.code + '] ' + language.name,
+            value: languageCode,
+            text: '[' + languageCode + '] ' + languages[languageCode].name,
           })
         );
-      });
-      $('#node-config-input-language').val(node.language);
+      }
+      $('#node-config-input-language').val(node.language || 'en-US');
```

Comment:
Can we move `en-US` to `language` property in `defaults`?

Reply:
Sure.